lib/conrad/emitters/stdout_emitter.rb

```diff
@@ -0,0 +1,11 @@
+module Conrad
+  module Emitters
+    # Basic emitter for sending events to $stdout.
+    class StdoutEmitter
```

Comment:
I dropped the emitter on mine because of the module name, it felt redundant to me and just made the name longer. I'd rather drop emitter/processer/whatevs from the names if it's all the same to you. Thoughts?

Reply:
Go for it! Just be sure to update the README at the same time as the relevant code :)